src/sparseml/pytorch/image_classification/utils/trainer/_trainer.py

```diff
@@ -0,0 +1,365 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
as discussed, let's move the `trainer` directory to just be trainer.py within utils